packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -0,0 +1,61 @@
+@import '../../mixins/links.css';
+:host {
+  display: block;
+}
+a[aria-expanded='true'] ~ #info {
+  padding-left: calc(2rem - 7px);
+  border-left: 7px solid transparent;
+  border-left-color: var(--color-primary-alt-light);
+}
+
+#info.closed {
+  visibility: hidden;
+}
```

Comment:
This fixes the problem that @k80bowman noticed. The reason it was happening with this web component and not with the older React component is because the React implementation wasn't rendering the children when the component was "closed":

https://github.com/department-of-veterans-affairs/component-library/blob/4a7c522fe2f21d8ba0e7b0bea795fe0d959a2a0c/packages/react-components/src/components/ExpandingGroup/ExpandingGroup.jsx#L37-L40

But with the Web Component implementation, that child content is rendered by default into the `<slot>`. `visibility: hidden` [removes the node and its child nodes from the accessibility tree](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns):

> Using a `visibility` value of `hidden` on an element will remove it from the accessibility tree. This will cause the element and all its descendant elements to no longer be announced by screen reading technology.

Reply:
This fix works great, thank you!